tests/pre_processing/test_extract_landscape.py

```diff
@@ -0,0 +1,14 @@
+import unittest
+from pre_processing.extract_landscape import get_facial_vectors
+import logging
+
+class TestExtractLandScape(unittest.TestCase):
```

Comment:
(im nitpicking here) dependency versions are handled by pip
only specifying what libraries are actually needed helps readability and understanding of dependencies of _our_ code (i.e. we never referenced decorator or dask)